js/web/script/test-runner-cli-args.ts

```diff
@@ -381,9 +387,8 @@ export function parseTestRunnerCliArgs(cmdlineArgs: string[]): TestRunnerCliArgs
     cpuOptions,
     cpuFlags,
```

Comment:
cpu flags can also be removed. this is a placeholder for flags similar to webglFlags and wasmFlags. Currently there is no any flag in it. if we want to add any, the plan is to add it into env like env.cpu.xxx